### PR TITLE
First readme documentation approach

### PR DESCRIPTION
**Pull request by trikunai**
First proposal with requirements, how to install, and how to run different projects from monorepo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,39 @@
 [![BCH compliance](https://bettercodehub.com/edge/badge/legalthings/lto-network-frontend?branch=master)](https://bettercodehub.com/)
 
-# LtoNetworkFrontend
+# LTO Network Frontend
+<a href="https://github.com/legalthings/lto-network-frontend/"><img src="https://lto.network/img/meta.png"></a>
+<!-- This project was generated with [Angular CLI](https://github.com/angular/angular-cli) using [Nrwl Nx](https://nrwl.io/nx).  -->
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) using [Nrwl Nx](https://nrwl.io/nx).
+## The Goal of Angular CLI
```
Comment (KillaBee1985) [inline README.md]: This is not a goal of angular CLI. I think it would be better to name it as "Repository content".

File: README.md [hunk 2]
```diff
@@ -1,10 +1,39 @@
...
+
+To get started locally, follow these instructions:
+If you haven't done it already, make a fork of this [repo](https://github.com/legalthings/lto-network-frontend/fork).
+1. Clone to your local computer using `git`.
+1. Make sure that you have Node 10.9 or later installed. See instructions [here](https://nodejs.org/en/download/). The Angular CLI requires Node 8, but development requires Node 10.
+1. Enter project main directory and install npm modules (dependecies): `npm install`.
+
+## Getting Started - Local Development
+
+1. If you want to run `explorer` project, you can just use `ng serve`.
```
Comment (KillaBee1985) [inline README.md]: To start explorer you need to use `ng serve explorer`. It just happened to be that explorer is a default project.

File: README.md [hunk 3]
```diff
@@ -18,11 +47,21 @@ Nx is designed to help you create and build enterprise grade Angular application
...
-Run `ng generate app myapp` to generate an application. When using Nx, you can create multiple applications and libraries in the same CLI workspace. Read more [here](http://nrwl.io/nx).
-
+Run `ng generate app myapp` to generate an application. When using Nx, you can create multiple applications and libraries in the same CLI workspace. Read more [here](http://nrwl.io/nx). -->
+<!-- 
 ## Development server
 
-Run `ng serve --project=myapp` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+Run `ng serve --project=myapp` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files. -->
+
+## The Goal of Angular CLI
```
Comment (KillaBee1985) [inline README.md]: I do not think we need to tell what is the goal of Angular CLI because it is not related to our project.

File: README.md [hunk 4]
```diff
@@ -18,11 +47,21 @@ Nx is designed to help you create and build enterprise grade Angular application
...
+
+The Angular CLI creates, manages, builds and test your Angular projects. It's built on top of the
+Angular DevKit.
+
+## The Goal of DevKit
```
Comment (KillaBee1985) [inline README.md]: Same here. What is `DevKti`?

File: README.md [hunk 5]
```diff
@@ -18,11 +47,21 @@ Nx is designed to help you create and build enterprise grade Angular application
...
+
+DevKit's goal is to provide a large set of libraries that can be used to manage, develop, deploy and
+analyze your code.
 
 ## Code scaffolding
```
Comment (KillaBee1985) [inline README.md]: It is not related to project so we can remove it.

File: README.md [hunk 6]
```diff
@@ -43,4 +82,4 @@ Before running the tests make sure you are serving the app via `ng serve`.
 
 ## Further help
```
Comment (KillaBee1985) [inline README.md]: I dont think that future help has something to do with angular cli.

**Comment by trikunai** [comment]
resolved, you were right !

**Pull request merged**